src/test/java/com/google/devtools/build/lib/query2/cquery/ProtoOutputFormatterCallbackTest.java

```diff
@@ -213,6 +217,25 @@ public void testConfigurations() throws Exception {
             .findAny()
             .orElseThrow();
     assertThat(fileTargetProto.getConfigurationId()).isEqualTo(0);
+
+    // Targets whose deps have no transitions should appear without configuration information.
+    assertThat(Iterables.getOnlyElement(parentRuleProto.getTarget().getRule().getConfiguredRuleInputList()))
+        .isEqualTo(ConfiguredRuleInput.newBuilder().setLabel("//test:transition_rule").build());
```

Comment:
Maybe this could be simplified to ?
```
assertThat(parentRuleProto.getTarget().getRule().getConfiguredRuleInputList())
        .containsExactly(
            ConfiguredRuleInput.newBuilder().setLabel("//test:transition_rule").build());
```